src/main/java/com/thealgorithms/strings/ReverseWordsInString.java

```diff
@@ -0,0 +1,25 @@
+package com.thealgorithms.strings;
+
+import java.util.Arrays;
+import java.util.Collections;
+
+/**
+ * Given an input string s, reverse the order of the words.
+ * A word is defined as a sequence of non-space characters.
+ * The words in s will be separated by at least one space.
+ * Return a string of the words in reverse order concatenated by a single space.
+ * Note that s may contain leading or trailing spaces or multiple spaces between two words.
+ * The returned string should only have a single space separating the words.Do not include any extra spaces
+ */
```

Comment:
Please update the description. Currently it is a description of the problem. Basically it could be removed.

Reply:
done